tiatoolbox/models/architecture/unet.py

```diff
@@ -256,62 +307,14 @@ def __init__(
         # channel mapping for shortcut
         self.conv1x1 = nn.Conv2d(down_ch_list[0], down_ch_list[1], (1, 1), bias=False)
 
-        def create_block(kernels, input_ch, output_ch):
-            """Helper to create a block of Vanilla Convolution.
-
-            This is in pre-activation style.
-
-            Args:
-                kernels (list):
-                    A list of convolution layers. Each item is an
-                    integer and denotes the layer kernel size.
-                input_ch (int):
-                    Number of channels in the input images.
-                output_ch (int):
-                    Number of channels in the output images.
-
-            """
-            layers = []
-            for ksize in kernels:
-                if preact:
-                    layers.extend(
-                        [
-                            nn.BatchNorm2d(input_ch),
-                            nn.ReLU(),
-                            nn.Conv2d(
-                                input_ch,
-                                output_ch,
-                                (ksize, ksize),
-                                padding=int((ksize - 1) // 2),  # same padding
-                                bias=False,
-                            ),
-                        ]
-                    )
-                else:
-                    layers.extend(
-                        [
-                            nn.Conv2d(
-                                input_ch,
-                                output_ch,
-                                (ksize, ksize),
-                                padding=int((ksize - 1) // 2),  # same padding
-                                bias=False,
-                            ),
-                            nn.BatchNorm2d(output_ch),
-                            nn.ReLU(),
-                        ]
-                    )
-                input_ch = output_ch
-            return layers
-
         self.uplist = nn.ModuleList()
         for ch_idx, ch in enumerate(down_ch_list[1:]):
             next_up_ch = ch
             if ch_idx + 2 < len(down_ch_list):
                 next_up_ch = down_ch_list[ch_idx + 2]
             if self.skip_type == "concat":
                 ch *= 2
-            layers = create_block(decoder_block, ch, next_up_ch)
+            layers = create_block(preact, decoder_block, ch, next_up_ch)
```

Comment:
What is 'preact'?

Reply:
If it is pre activation, could we just call it that, i.e. `pre_activation`? To make it more readable.

Reply:
Resolved in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/345/commits/393fe26d121ef21f28b56e0894bf14e69e98b0ec